src/emqx_connection.erl

```diff
@@ -589,6 +589,11 @@ parse_incoming(Data, Packets, State = #state{parse_state = ParseState}) ->
             NState = State#state{parse_state = NParseState},
             parse_incoming(Rest, [Packet|Packets], NState)
     catch
+        error:proxy_protocol_config_disabled:_Stk ->
```

Comment:
Do not match stacktrace if not needed.
It’s costs

Reply:
Ignored var still costs?

Reply:
Yes